server/views/makeAReferral/completionDeadline.njk

```diff
@@ -19,6 +20,10 @@
 
         {{ govukDateInput(dateInputArgs) }}
 
+        {% if sentReferral %}
```

Comment:
you can do without the additional logic for sent referral in this template and just render the input if it's present 

so `{% if textAreaArgs %}... `